### PR TITLE
refactor code 👍🏻 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -49,67 +49,66 @@ async fn main() {
...
-                    mouse_position().1,
-                    BRUSH.size,
+                    dot.x,
+                    dot.y,
+                    dot.size,
                     macroquad::color::Color::from_rgba(
-                        (BRUSH.r * 255.0) as u8,
-                        (BRUSH.g * 255.0) as u8,
-                        (BRUSH.b * 255.0) as u8,
+                        (dot.r * 255.0) as u8,
```
Comment (ateschan) [inline src/main.rs:56]: Use the method on BRUSH

**Comment by ateschan** [comment]
merging into feature for a few edits

**Comment by ghost** [comment]
Its already using BRUSH. 

The dot struct above

r = BRUSH.r

so we do dot.r and so on with the rest 
IMG_7522

**Comment by ateschan** [comment]
I see what you were going for there. My code was wrong before your edit, that draw_circle function should be outside the if statement but within the unsafe block so it shows a little circle on the cursor.

**Pull request merged**